### ISSUE TITLE
Plot save modal keyboard navigation not working as expected

### ISSUE THREAD

**Issue by jonvanausdeln** [posit-dev/positron#6121]
#### Positron and OS details:

Positron Version: 2025.02.0 (system setup) build 112
Code - OSS Version: 1.96.0
Commit: 5868434c4f9757d050221b707eb2fdc3a6284dee
Date: 2025-01-24T03:40:16.944Z
Electron: 32.2.6
Chromium: 128.0.6613.186
Node.js: 20.18.1
V8: 12.8.374.38-electron.0
OS: Windows_NT x64 10.0.26100

#### Interpreter details:

N/A

## Describe the issue:

See: https://github.com/posit-dev/positron/issues/3548#issuecomment-2612917536

At some point the way keyboard navigation worked in the plot save dialog has changed. And not all the fields are accessible with just the keyboard

I found that tabbing around the save plot dialog isn't working as it used to.  It only cycles between directory, browse button, and name field.  It used to cycle through all the fields.

Old behavior (version 2024.11.0-140): 

https://github.com/user-attachments/assets/e3e6d75f-5166-4e8f-be85-7dc5644c5a9b

New behavior:

https://github.com/user-attachments/assets/8a1a112c-6fdb-4ec1-ab6c-2de59ffe34ea

## Expected or desired behavior:
Tab goes to all the elements

**Comment by bricestacey** [comment]
Using the official releases:

I can reproduce using :

Positron Version: 2025.01.0 (Universal) build 159
Code - OSS Version: 1.95.0
Commit: bc8820be811f8697ae06e8fda469be39b5fd0f16
Date: 2025-01-09T23:29:00.471Z
Electron: 32.2.1
Chromium: 128.0.6613.186
Node.js: 20.18.0
V8: 12.8.374.38-electron.0
OS: Darwin arm64 25.1.0

I *cannot* reproduce using:

Positron Version: 2024.12.0 (Universal) build 96
Code - OSS Version: 1.93.0
Commit: c5ce275dc502f6b15433b271802cb33e1ba5ef68
Date: 2024-11-28T02:47:29.898Z
Electron: 30.4.0
Chromium: 124.0.6367.243
Node.js: 20.15.1
V8: 12.4.254.20-electron.0
OS: Darwin arm64 25.1.0

Notable but should not be relevant -- between these two versions, the order of panes swapped. With 2024.12, the plot is in the top right. With 2025.01, the plot moves to the bottom right.
